intrusion-prevention/js/MainController.js

```diff
@@ -156,10 +222,15 @@ Ext.define('Ung.apps.intrusionprevention.MainController', {
         // console.log(signatures.length);
 
         // Process custom signatures
-        vm.get('settings.signatures.list').forEach(function(settingsSignature){
-            signatures.push(new Ung.model.intrusionprevention.signature(settingsSignature.signature, settingsSignature.category, false));
-        });
+        // vm.get('settings.signatures.list').forEach(function(settingsSignature){
+        //     signatures.push(new Ung.model.intrusionprevention.signature(settingsSignature.signature, settingsSignature.category, false));
+        // });
+        // console.log("set signaturesList with " + signatures.length);
```

Comment:
Is there a reason to leave this code, but commented out?

Reply:
Good catch!  Thanks!